lib/routers/establishment/invite-user.js

```diff
@@ -33,7 +33,7 @@ const preventDuplicateInvite = (req, res, next) => {
     .then(() => {
       return req.models.Profile.query()
         .scopeToEstablishment('establishments.id', req.establishment.id)
-        .where('profiles.email', req.body.data.email)
+        .whereRaw('LOWER(profiles.email) LIKE ?', [req.body.data.email.toLowerCase()])
```

Comment:
You can use an `iLike` here instead without needing to break into raw query building.

```suggestion
        .where('profiles.email', 'iLike', req.body.data.email)
```